seacrowd/sea_datasets/newsph/newsph.py

```diff
@@ -0,0 +1,128 @@
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@article{cruz2020investigating,
+  title={Investigating the True Performance of Transformers in Low-Resource Languages: A Case Study in Automatic Corpus Creation},
+  author={Jan Christian Blaise Cruz and Jose Kristian Resabal and James Lin and Dan John Velasco and Charibeth Cheng},
+  journal={arXiv preprint arXiv:2010.11574},
+  year={2020}
+}
+"""
+_DATASETNAME = "newsph"
+_LANGS = ["fil", "tgl"]
+_DESCRIPTION = """\
+Raw collection of news articles in Filipino which can be used for language modelling.
+"""
+_HOMEPAGE = "https://huggingface.co/datasets/newsph"
+_LICENSE = Licenses.GPL_3_0.value
+_LOCAL = False
+_URLS = "https://s3.us-east-2.amazonaws.com/blaisecruz.com/datasets/newsph/newsph-nli.zip"
+_SUPPORTED_TASKS = [Tasks.TEXTUAL_ENTAILMENT]
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class NewsPhDataset(datasets.GeneratorBasedBuilder):
+    """
+    Raw collection of news articles in Filipino which can be used for language modelling.
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name="newsph_source",
+            version=SOURCE_VERSION,
+            description="newsph source schema",
+            schema="source",
+            subset_id="newsph",
+        ),
+        SEACrowdConfig(
+            name="newsph_seacrowd_pairs",
+            version=SEACROWD_VERSION,
+            description="newsph SEACrowd schema",
+            schema="seacrowd_pairs",
+            subset_id="newsph",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "newsph_source"
```

Comment:
I don't know if we should enforce this but other reviewers seem to favor the usage of _DATASETNAME constant in configs even though it is not explicitly prescribed in the guidelines. Personally I'm okay with this and I think it's cleaner this way.

Reply:
I'm okay with this as well as long as the output is correct, so no problem.